components/treeview/stories/treeview.stories.js

```diff
@@ -281,20 +453,25 @@ WithDropTarget.args = {
 		},
 	],
 };
-WithDropTarget.tags = ["autodocs", "!dev"];
-WithDropTarget.parameters = {
+DropTarget.tags = ["autodocs", "!dev"];
+DropTarget.parameters = {
 	chromatic: { disableAllSnapshots: true },
 };
 
+/**
+ * A tree view drawn without nesting, suitable for infinite scrolling. With this version of the treeview, you must manage the visibility of "child items" manually based on the open state of the "parent item." The level of visual indentation is handled by a numbered `indent` variant class.
+ */
 export const Flat = Template.bind({});
 Flat.storyName = "Flat Markup";
 Flat.args = {
+	customStyles: {
+		"width": "500px",
+	},
```

Comment:
```suggestion
	customStyles: {
		"max-inline-size": "500px",
	},
```
Looking at the docs page, I see the Flat example shows me truncated text on the menu item that contains "This example has longer text...". But in the next example for "Folders and Files", the item does not truncate. As a user I might think the variants handle this differently and might not notice the inline style setting the width.

Would it be worth surfacing this in the description, and setting the same behavior for both examples? For example, "This example sets a maximum width of 500px in order to demonstrate how long text truncates".

Reply:
Ope. Yep, I changed this to `max-inline-size` and applied it to both Flat and "Folders and Files", and mentioned that it was meant to be truncated by setting that in both stories. I also moved the stories around so that it would make sense in the code, too, not just when the Storybook shows the alphabetized stories.

Reply:
Thanks! This looks great.